drivers/couchbase/couchbase-driver/src/main/java/io/mongock/driver/couchbase/util/CouchbaseVersionUtil.java

```diff
@@ -0,0 +1,55 @@
+package io.mongock.driver.couchbase.util;
+
+import com.couchbase.client.core.deps.com.fasterxml.jackson.databind.JsonNode;
+import com.couchbase.client.core.json.Mapper;
+import com.couchbase.client.core.manager.CoreBucketManager;
+import com.couchbase.client.java.AsyncUtils;
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.manager.bucket.GetBucketOptions;
+
+public final class CouchbaseVersionUtil {
```

Comment:
I think this file is useless if we don't validate server in the startup, but if would be useful in if the near future we'll decide to check it. So it's fine.

Reply:
Yep, let's keep it for now, in case we do not need it we can just get rid of it whenever we want. There should be almost 0 effort to maintain this "util".